webapp/src/components/sidebar_right/index.tsx

```diff
@@ -127,6 +127,7 @@ function SidebarRight({theme}: {theme: Theme}) {
         };
     });
 
+    // States used for storing the updated details of PRs, reviews, and RHSState after being taken from the redux store.
     const [updatedPrs, setUpdatedPrs] = useState<Item[]>(yourPrs);
     const [RHSState, setRHSState] = useState('');
```

Comment:
Add a separate comment for this state variable as this is only being used to denote the previous RHS state. And also rename this - 
```suggestion
    const [previousRhsState, setPreviousRhsState] = useState('');
```

Reply:
Removed this state